src/lib/libdll/meson.build

```diff
@@ -0,0 +1,14 @@
+libdll_files = ['dlfcn.c', 'dllcheck.c',  'dllerror.c',  'dllfind.c', 'dll_lib.c',  'dlllook.c',  'dllnext.c',  'dllopen.c',  
+                'dllplug.c', 'dllscan.c']
+
+libdll_c_args = shared_c_args + [
+    '-D_BLD_Dll=1',
```

Comment:
The `=1` isn't needed and isn't consistent with all the other true/false symbols we define in the build system.